src/inspector/models/text/textsettingsmodel.cpp

```diff
@@ -111,8 +112,8 @@ void TextSettingsModel::loadProperties()
     m_fontStyle->setIsEnabled(true);
 
     loadPropertyItem(m_fontSize, [](const QVariant& elementPropertyValue) -> QVariant {
-        return elementPropertyValue.toInt() == mu::engraving::TextBase::UNDEFINED_FONT_SIZE
-               ? QVariant() : elementPropertyValue.toInt();
+        return RealIsEqual(elementPropertyValue.toDouble(),double(mu::engraving::TextBase::UNDEFINED_FONT_SIZE))
```

Comment:
I meant changing the _definition_ of `UNDEFINED_FONT_SIZE` to double.

Reply:
OK